### PR TITLE
Cumulative Fee Calculation for Gemini Orders

### PR DESCRIPTION
**Pull request by gmax0**
Removed some boilerplate POJO + added UserReference to the limit order object returned from the getOrderDetails call.

### CODE REVIEW

File: xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java [hunk 1]
```diff
@@ -122,6 +122,19 @@ public static Order adaptOrder(GeminiOrderStatusResponse geminiOrderStatusRespon
     OrderStatus orderStatus = adaptOrderstatus(geminiOrderStatusResponse);
     Date timestamp = new Date(geminiOrderStatusResponse.getTimestampms());
 
+    // Calculate Fees in counter currency
+    BigDecimal fee = null;
```
Comment (earce) [inline xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java:126]: set this to BigDecimal.ZERO and then in your loop you can just add for all and it becomes a 1-liner

File: xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/service/GeminiTradeServiceRaw.java [hunk 2]
```diff
@@ -134,6 +132,47 @@ public GeminiOrderStatusResponse getGeminiOrderStatus(String orderId) throws IOE
...
+      if (orderId == null) {
+        clientOrderId = ((GeminiTradeService.GeminiOrderQueryParams) params).getClientOrderId();
+      }
+
+      includeTrades = ((GeminiTradeService.GeminiOrderQueryParams) params).isIncludeTrades();
+      account = ((GeminiTradeService.GeminiOrderQueryParams) params).getAccount();
+    }
+
+    try {
+      GeminiOrderStatusResponse orderStatus =
```
Comment (earce) [inline xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/service/GeminiTradeServiceRaw.java:154]: just return inline, no need to set this var

File: xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java (hunk 1 above)
Comment (gmax0) [reply xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java:126]: I ran into issues with the scale of the BigDecimal when setting to BigDecimal.ZERO originally.
Example, adding 0 to 0.00023 would still result in a BigDecimal displaying as 0. Tried to address this by setting the scale before and after the calculations, but it didn't seem to work.

Comment (gmax0) [reply xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/GeminiAdapters.java:126]: I was debugging the BigDecimal returned using IntelliJ's debugger.

File: xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/service/GeminiTradeServiceRaw.java (hunk 2 above)
Comment (gmax0) [reply xchange-gemini/src/main/java/org/knowm/xchange/gemini/v1/service/GeminiTradeServiceRaw.java]: Done

**Pull request merged**